dhis-2/dhis-services/dhis-service-setting/src/main/java/org/hisp/dhis/setting/SystemSettingManager.java

```diff
@@ -175,7 +175,7 @@ default Boolean getBooleanSetting( SettingKey key )
     @Transactional( readOnly = true )
     default boolean getBoolSetting( SettingKey key )
     {
-        return getSystemSetting( key, Boolean.class ) == Boolean.TRUE;
+        return getSystemSetting( key, Boolean.class ) == Boolean.TRUE.booleanValue();
```

Comment:
I think this is good. Though it seems we get an object Boolean back from `getSystemSetting`, so by doing == boolean primitive we rely on auto-boxing for this to work. Would this be a slightly better approach, ` Boolean.TRUE.equals` will compare object Booleans and also tolerate nulls?

`return Boolean.TRUE.equals( getSystemSetting( key, Boolean.class ) );`

Reply:
Agree, using `equals()` is a cleaner approach. Fixed now.